File: PHP/Ram.php
<?php
class Ram
{
    private $capacity;
    private $price;

    function __construct($capacity = "", $price = 0)
    {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    function setCapacity($capacity)
    {
        $this->capacity = $capacity;
    }

    function getCapacity()
    {
        return $this->capacity;
    }

    function setPrice($price)
    {
        $this->price = $price;
    }

    function getPrice()
    {
        return $this->price;
    }

    function showRam()
    {
        echo "RAM Capacity    : " . $this->getCapacity() . "<br>";
        echo "RAM Price       : " . $this->getPrice() . "<br>";
    }

    function __destruct(){}
}

File: java/Ram.java
public class Ram
{
    private String capacity = "";
    private int price = 0;

    Ram(){}

    Ram(String capacity, int price)
    {
        this.capacity = capacity;
        this.price = price;
    }

    public void setCapacity(String capacity)
    {
        this.capacity = capacity;
    }

    public String getCapacity()
    {
        return this.capacity;
    }

    public void setPrice(int price)
    {
        this.price = price;
    }

    public int getPrice()
    {
        return this.price;
    }

    public void showRam()
    {
        System.out.println("RAM Capacity    : " + this.getCapacity());
        System.out.println("RAM Price       : " + this.getPrice());
    }
}